src/components/StoriesInsights/useStyles.js

```diff
@@ -0,0 +1,103 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(
+  ({ palette, typography, breakpoints, widths }) => ({
+    root: {
+      backgroundColor: "#F8F8F8",
+    },
+    section: {
+      paddingTop: typography.pxToRem(60),
+      paddingLeft: typography.pxToRem(20),
+      paddingRight: typography.pxToRem(20),
```

Comment:
[`Section`](https://github.com/CodeForAfrica/PesaYetu/blob/d4fa0b1033d5ab829f78046618819174d989f222/src/components/Section/index.js#L9) already defines left/right padding of 20px.